test/functional/test-ads-config.js

```diff
@@ -18,6 +18,17 @@ import {adConfig} from '../../ads/_config';
 
 describe('test-ads-config', () => {
 
+  it('should have all ad networks configured', () => {
+    window.ampTestRuntimeConfig.adTypes.forEach(adType => {
```

Comment:
nah, I think about it. probably not since we already have code review. This is good